docs/plugins/backend-plugin.md

```diff
@@ -159,24 +158,36 @@ database..
 
 ## Making Use of the User's Identity
 
-The Backstage backend comes with a facility for retrieving the identity of the
-logged in user.
-
-As part of the environment object that is passed to your `createPlugin`
-function, there is a `identity` field. You can use that to get an identity
-from the request.
-
-```ts
-// in packages/backend/src/plugins/carmen.ts
-export default async function createPlugin(
-  env: PluginEnvironment,
-): Promise<Router> {
-  return await createRouter({
-    model: model,
-    logger: env.logger,
-    identity: env.identity,
-  });
-}
+The Backstage backend also offers a core service to access the user's identity. You can access it through the `coreServices.identity` dependency.
```

Comment:
mmh all of this is being replaced by `coreServices.httpAuth` and `coreServices.auth`

also maybe it's worthwhile making one single code example for how to _generally_ add `coreServices` deps? and then link to eg https://backstage.io/docs/backend-system/core-services/http-auth (pending to be filled in soon!) for details on how it works and what it does?

Reply:
Gotcha, what should this be then? `coreServices.auth` with a secondary call for things like ownership references?

Yup, I'll add something today.

Reply:
Ooo so new! Added a block for the how to add dependencies as well as the current set of core services.

Reply:
Hey I unresolved this, sorry - `identity` will be deprecated very soon, if you want an example for getting the incoming credentials you could make one that uses `coreServices.httpAuth` and `coreServices.userInfo` to get the identity and ownership refs.

Reply:
whoops 😅 , updated that line as well.